api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -173,7 +170,7 @@ public static String getOpenmrsProperty(String property) {
 	 * The name of the runtime property that a user can set that will specify whether the database
```

Comment:
the APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY can also be `static final`

Reply:
@teleivo I made the necessary changes. Feel free to give feedback. Thanks